tools/postgres/Dockerfile

```diff
@@ -9,7 +9,7 @@ COPY certs/postgres.server.crt /var/lib/postgresql/postgres.server.crt
 COPY certs/postgres.server.key /var/lib/postgresql/postgres.server.key
 RUN chown -R postgres /var/lib/postgresql && \
 	chmod 600 /var/lib/postgresql/postgres.server.key
-CMD ["-c", "ssl=on", "-c", "ssl_cert_file=/var/lib/postgresql/postgres.server.crt", "-c", "ssl_key_file=/var/lib/postgresql/postgres.server.key", "-c", "ssl_ca_file=/var/lib/postgresql/rootCA.crt", "-c", "hba_file=/var/lib/postgresql/pg_hba.conf"]
+CMD ["-c", "ssl=on", "-c", "max_connections=10000", "-c", "ssl_cert_file=/var/lib/postgresql/postgres.server.crt", "-c", "ssl_key_file=/var/lib/postgresql/postgres.server.key", "-c", "ssl_ca_file=/var/lib/postgresql/rootCA.crt", "-c", "hba_file=/var/lib/postgresql/pg_hba.conf"]
```

Comment:
as explained in the description, bumping the connection limit makes the tests pass locally

Reply:
This only happens on `tls` postgres right? What was the default max_connections btw?

Reply:
We're not recreating the store 59 times normally but I agree and I'll change that :)

Reply:
done